### PR TITLE
Added LC 1312 and updated the README.md

### PR DESCRIPTION
**Pull request by godmodevegeta**

### CODE REVIEW

**Review by Jiganesh — CHANGES_REQUESTED** [review]
The problem is solved under dynamic programming folder.
Please add link to the problem at top of java solution file and put java file inside dynamic programming folder.
remove link added in readme file as it is already present in dynamic programming section

**Comment by godmodevegeta** [comment]
Hi, based on your suggestions, I've
- Added link to the problem on top of the solution file
- Moved the file from string to dynamicProgramming
- Deleted the link from README.md

File: README.md [hunk 1]
```diff
@@ -574,7 +574,6 @@
 <details>
 <summary>Strings</summary>
```
Comment (godmodevegeta) [inline README.md:576]: I initially thought about adding a different problem I practised today. But then just did what you suggested.

**Comment by Jiganesh** [comment]
Thank you !!

**Pull request merged**